site/content/how-to/monitoring/troubleshooting.md

```diff
@@ -65,54 +65,297 @@ LAST SEEN   TYPE      REASON              OBJECT
 5s          Warning   ResourceDeleted     nginxgateway/ngf-config                          NginxGateway configuration was deleted; using defaults
 ```
 
+##### Get shell access to containers
+
+Getting shell access to containers allows developers and operators to view the environment of a running container, see its logs or diagnose any problems. To get shell access to the NGINX container, use `kubectl exec`:
+
+```shell
+kubectl exec -it [-n namespace] <ngf-pod-name> -c nginx /bin/sh
+```
+
 ##### Logs
 
 Logs from the NGINX Gateway Fabric control plane and data plane can contain information that isn't available to status or events. These can include errors in processing or passing traffic.
 
-To see logs for the control plane container:
+1. To see logs for the control plane container:
```

Comment:
Is this `1.` needed? There isn't a `2.` that follows

Reply:
I was following the ordered number list connotation as used in our other docs, but have updated it to 
1. Logs levels
2. errors logs
3. access logs

Reply:
was running into lint errors due to `1. , 2. 3.` numbered style 

```
site/content/how-to/monitoring/troubleshooting.md:108:1 MD029/ol-prefix Ordered list item prefix [Expected: 1; Actual: 2; Style: 1/1/1]
site/content/how-to/monitoring/troubleshooting.md:137:1 MD029/ol-prefix Ordered list item prefix [Expected: 1; Actual: 3; Style: 1/1/1]
```

Reply:
My bad, I didn't realize there were multiple items in the list. I thought the list contained just one item. All `1s` is fine